src/Nordea/Components/NumberInput.elm

```diff
@@ -141,6 +147,10 @@ getStyles config =
     in
     [ fontSize (rem 1)
     , height (rem 3)
+    , pseudoElement "-webkit-outer-spin-button" [ display none ]
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-outer-spin-button

Looks rough when it comes to compatibility. Are there any other solutions?

Reply:
That's why we also have this for firefox :) `property "-moz-appearance" "textfield"`

Reply:
I have at least tested in chrome, safari and mozilla and it hides it fine